src/Data/Map/Monoidal.hs

```diff
@@ -229,6 +230,13 @@ instance (Ord k, Semigroup a) => IsList.IsList (MonoidalMap k a) where
     {-# INLINE toList #-}
 #endif
 
+instance (Ord k) => Align (MonoidalMap k) where
```

Comment:
This can just be derived with GND.

Reply:
Agreed, these really should be derived.

However, I'm also quite concerned that adds yet more packages to this library's already large transitive closure.